package.json

```diff
@@ -22,22 +22,16 @@
     "cordova-ios",
     "cordova-android"
   ],
-  "engines": {
-    "cordovaDependencies": {
-        "<2.0.0": {
-          "cordova": ">=3.5.0"
-        },
-        "2.0.0": {
-          "cordova": ">=6.1.0",
-          "cordova-android": ">=6.1.0",
-          "cordova-ios": ">=4.3.1"
-        }
-    }
-  },
   "author": "xu.li <AthenaLightenedMyPath@gmail.com>",
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/xu-li/cordova-plugin-wechat/issues"
   },
-  "homepage": "https://github.com/xu-li/cordova-plugin-wechat#readme"
+  "homepage": "https://github.com/xu-li/cordova-plugin-wechat#readme",
+  "scripts": {
+    "capacitor:sync:before": "node ./scripts/android-install.js"
```

Comment:
This will get executed automatically in Capacitor v6.1+, but not lesser versions. I have an [open PR](https://github.com/quanticedu/back_royal/pull/13992) to put back_royal on 6.1.2, but am working through some pesky issues there, so decided to also export this as a `bin` script and just call it from a `capacitor:sync:before` hook in `hybrid/quantic/package.json` for now (seems to work fine).